### ISSUE TITLE
Autohide Tabs

### ISSUE THREAD

**Issue by Tomoghno** [rafaelmardojai/firefox-gnome-theme#274]
autohiding Tabs will be a Sweet Addition

**Comment by rafaelmardojai** [comment]
To my knowledge this isn't a pattern on GNOME tabs. So it's kinda out of scope. 

Or maybe I'm miss understanding what you mean with autohide tabs.

**Comment by Tomoghno** [comment]
> To my knowledge this isn't a pattern on GNOME tabs. So it's kinda out of scope.
> 
> Or maybe I'm miss understanding what you mean with autohide tabs.

Yeah i forgot the fact that it should match GNOME Human Design Guidelines.
